test/core/end2end/BUILD

```diff
@@ -679,3 +453,6 @@ grpc_cc_proto_library(
     name = "end2end_test_fuzzer_cc_proto",
     deps = ["end2end_test_fuzzer_proto"],
 )
+
+grpc_core_end2end_test_suite(name = "end2end",
+config_src = "end2end_config.cc")
```

Comment:
Weird formatting here.  I assume sanity will catch this, but thought I'd point it out just in case.